internal/projectroot/path.go

```diff
@@ -12,28 +12,16 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package providers
+package projectroot
 
 import (
-	"k8s.io/apimachinery/pkg/runtime/schema"
+	"path/filepath"
+	"runtime"
 )
 
-type ProviderAzure struct{}
+var (
+	_, b, _, _ = runtime.Caller(0)
```

Comment:
this is not a stable solution (it works and I am not aware of another approach) in terms of the file location, so i'd explicitly comment on the relative file path in which this particular file should be placed, namely `internal/projectroot/path.go` and it is not recommended to move the file to another path